KsApi/MockService.swift

```diff
@@ -514,6 +515,19 @@ internal struct MockService: ServiceType {
     )
   }
 
+  func fetchGraphCategories(query: NonEmptySet<Query>) -> SignalProducer<RootCategoriesEnvelope, GraphError> {
+    return SignalProducer(value: self.fetchGraphCategoriesResponse!)
+  }
+
+  func fetchGraphCategory(query: NonEmptySet<Query>)
+    -> SignalProducer<RootCategoriesEnvelope.Category, GraphError> {
+    return SignalProducer(value: .template |> RootCategoriesEnvelope.Category.lens.id .~ "\(query.head)")
+  }
+
+  internal func fetchGraph<A>(query: NonEmptySet<Query>) -> SignalProducer<A, GraphError> where A: Decodable {
+    return .empty
```

Comment:
If we never need to mock this then we could perhaps remove it from the interface and always create query specific functions?

Reply:
Oh right, I will remove it. Should have done this in the past. Good catch!